strawberry/asgi/__init__.py

```diff
@@ -48,6 +46,12 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send):
         else:
             raise ValueError("Unknown scope type: %r" % (scope["type"],))
 
+    async def get_root_value(self, request: Request) -> typing.Optional[typing.Any]:
```

Comment:
Note: I made this (and `get_context`) async because it might be useful. Not sure if it has any downsides though.